staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/generated.proto

```diff
@@ -51,7 +51,8 @@ message ConversionResponse {
 
   // `convertedObjects` is the list of converted version of `request.objects` if the `result` is successful otherwise empty.
   // The webhook is expected to set apiVersion of these objects to the ConversionRequest.desiredAPIVersion. The list
-  // must also has the same size as input list with the same objects in the same order(i.e. equal UIDs and object meta)
+  // must also have the same size as the input list, with the same objects, in the same order (i.e. equal UIDs, name, namespace)
+  // and the same metadata with the exception of labels and annotations. Any other mutated fields in metadata are ignored.
```

Comment:
Are we saying all metadata fields except labels and annotations must be unchanged, or if they are changed, the field values will be ignored?

Reply:
this was trying to say the webhook could change labels/annotations

Reply:
Got it, thanks for clarifying

Reply:
Improved the wording.